libr/include/r_util/r_num.h

```diff
@@ -51,6 +51,21 @@ typedef const char *(*RNumCallback2)(struct r_num_t *self, ut64, int *ok);
 
 R_API RNum *r_num_new(RNumCallback cb, RNumCallback2 cb2, void *ptr);
 R_API void r_num_free(RNum *num);
+/**
+ * \brief Convert size in bytes to human-readable string (upto Exabyte)
```

Comment:
Don't put comments in include files. i dont think you need 15 lines to explain what the function does. Im not a big fan of comments as you can see :P and in fact the comments were suposed to be added only in the vapi files, so we can use valadoc to get documentation. for all the bindings at the same time, not just C.